website/docs/language/resources/provisioners/local-exec.mdx

```diff
@@ -39,8 +39,12 @@ The following arguments are supported:
 
 * `command` - (Required) This is the command to execute. It can be provided
   as a relative path to the current working directory or as an absolute path.
-  It is evaluated in a shell, and can use environment variables or Terraform
-  variables.
+  It is evaluated in a shell, and as such can use environment variables for
+  variable substitution. While Terraform variables may also be used for variable
+  substitution, doing so should be avoided as it can lead to [shell injection](https://en.wikipedia.org/wiki/Code_injection#Shell_injection)
+  vulnerabilities. Instead, one should pass Terraform variables to a command
+  through the `environment` parameter and use environment variable substitution
+  instead.
```

Comment:
```suggestion
  The `command` is is evaluated in a shell and can use environment variables for
  variable substitution. We do not recommend using Terraform variables for variable
  substitution because doing so can lead to shell injection vulnerabilities. Instead, you should pass Terraform variables to a command
  through the `environment` parameter and use environment variable substitution
  instead. Refer to the following OWASP article for additional information about injection flaws: [Code Injection](https://owasp.org/www-community/attacks/Code_Injection).
```
I _think_ OWASP might be a more authoritative reference than Wikipedia. All other suggestions, such as using active voice, are related to our style guide.